.vscode/extensions.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
probably worth deleteing any files under .vscode and pushing that change, so they're not in source control